styleguide/source/_patterns/03-organisms/four-up-teaser.json

```diff
@@ -0,0 +1,70 @@
+{
+  "items": [
+    {
+      "patternLab": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
```

Comment:
This should be `ama__link--black` since black is a modifier. Also I think black is the default so you may not need the class at all.